packages/flow-core/src/components/f-select/f-select.ts

```diff
@@ -507,6 +510,19 @@ export class FSelect extends FRoot {
 			bubbles: true,
 			composed: true
 		});
+		if (this.isStringsArray(this.selectedOptions as string[])) {
+			const event = new CustomEvent<FSelectCustomEvent>("input", {
+				detail: {
+					value: this.searchValue
+				},
+				bubbles: true,
+				composed: true
+			});
+			this.dispatchEvent(event);
+			(this.selectedOptions as string[]).push(this.searchValue);
+		}
+		this.openDropdown = false;
```

Comment:
please add these 2 lines in if block above

`this.openDropdown = false;
		this.clearFilterSearchString();`